native/packages/react-native-bpk-component-horizontal-nav/package-lock.json

```diff
@@ -0,0 +1,245 @@
+{
+  "name": "react-native-bpk-component-horizontal-nav",
```

Comment:
Have a look at all the other `package-lock.json` files ... non of them contain a `name` or `version` prop. I wonder what it is that is different between you environment and mines that causes this different behaviour. 

See https://github.com/Skyscanner/backpack/pull/229#discussion_r149375120 as a similar issue.

Also cc @matteo-hertel

Reply:
so turned out that on a fresh build (`npm run clean && npm i`) the name fields does not get added, I suspect this found it's way in as a weird combination of copy/paste, lerna and npm install